src/confirmation/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from 'react';
+
+import ButtonGroup from '../button-group';
+import Button      from '../button2';
+import SaveButton  from '../save-button';
+
+import Form        from '../../infl-components/form.jsx';
+import Tabs        from '../../infl-components/tabs.jsx';
+
+import Modal       from '../modal_dialog';
+import ModalHeader from '../modal_dialog/header';
+import ModalBody   from '../modal_dialog/body';
+import ModalFooter from '../modal_dialog/footer';
+
+export default class Confirmation extends Component {
+  static propTypes = {
+    body:     PropTypes.string,
+    title:    PropTypes.string,
+    no:       PropTypes.string,
+    yes:      PropTypes.string,
+    onYes:    PropTypes.func,
+    onNo:     PropTypes.func,
+    onCancel: PropTypes.func
+  };
+
+  static defaultProps = {
+    title:      'Are you sure?',
+    yes:        'OK',
+    no:         'Cancel',
+    onCancel:   null,
+    onNo:       function() {},
+    saveStatus: 'unsaved'
+  };
+
+  render() {
+    return (
+      <Modal size='small'>
+        <ModalHeader title={this.props.title} />
```

Comment:
not sure what you mean, I'm just referring to doing

```javascript
const { title, onNo, onCancel } = this.props;
```

instead of typing out this.props everywhere. We do that anytime there are more than 2 props usually.